### PR TITLE
import Antora component

### PR DESCRIPTION
**Pull request by mojavelinux**
- add Antora descriptor (antora.yml)
- set component name to elasticsearch-connector
- set component title to Elasticsearch Transport Plug-in
- update GitHub repository URLs
- add navigation file
- import pages and images

### CODE REVIEW

**Comment by mojavelinux** [comment]
Before merging, please confirm the name and version of the documentation component. These appear in the URL.

**Comment by dnault** [comment]
Can you please change the component name to "elasticsearch-connector"? (Thanks for calling this out!)

The component version (3.0) is correct.

**Comment by mojavelinux** [comment]
Indeed I can! Exactly the clarification I was looking for.

File: docs/modules/ROOT/pages/advanced-settings.adoc [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+* *couchbase.typeSelector*: Specifies the type selector class to be used for mapping documents to types.
+The following options are available:
+ ** `org.elasticsearch.transport.couchbase.capi.DefaultTypeSelector`: Maps all documents to the specified type (see immediately below).
+This is the default type selector, and so need not be explicitly specified.
+Type-specification is managed by the following settings:
+  *** *couchbase.typeSelector.defaultDocumentType*: The document type to which the `DefaultTypeSelector` maps all documents.
+The default is `couchbaseDocument`.
+  *** *couchbase.typeSelector.checkpointDocumentType*: The document type to which replication-checkpoint documents are mapped.
+The default is `couchbaseCheckpoint`.
+ ** `org.elasticsearch.transport.couchbase.capi.DelimiterTypeSelector`: If the document ID is in the format `<`_type_`><`_delimiter_`><*>`, documents are mapped to the type specified by `<`_type_`>`.
```
Comment (dnault) [inline docs/modules/ROOT/pages/advanced-settings.adoc]: When viewing the document on Github, this line doesn't render correctly. It shows up as:

    If the document ID is in the format <`type><`delimiter`><*>`,
    documents are mapped to the type specified by <`type>`. 

Does the same thing happen when the docs are processed by Antora, or does it look okay?

**Comment by mojavelinux** [comment]
Indeed. The converter struggled with that formatting sequence. I just pushed an update to fix it.

**Pull request merged**